spec/unit/installer/xcode/pods_project_generator/target_installer_spec.rb

```diff
@@ -54,7 +54,7 @@ class PodsProjectGenerator
             @pod_target.stubs(:uses_swift?).returns(true)
             @installer.send(:add_target)
             @installer.send(:native_target).resolved_build_setting('SWIFT_OPTIMIZATION_LEVEL').should == {
-              'Release' => nil,
+              'Release' => '-Owholemodule',
```

Comment:
To be clear: this is only for release builds. 

Also, the speed improvements can be massive, and to quote Apple: "the Swift Package Manager compiles with whole-module optimizations in release builds.". Seems pretty important to me for a dependency manager to build your dependencies in the most optimized way 🙂.